src/LevelCreator.cpp

```diff
@@ -0,0 +1,114 @@
+#include "LevelCreator.h"
+
+
+/* 
+Level Creator class serves as an engine for reading in a specific level text file
+and loading the proper tiles for that level to display to the player.
+*/
+
+LevelCreator::LevelCreator() {
+}
+
+void LevelCreator::init(){
+
+}
+
+void LevelCreator::LoadLevel(int level){
+    //This line will change based on "level", for now just using MapTest
+    // std::ifstream openfile("../res/MapTest.txt");
+
+    //This line will change based on "level", for now just using Sample Level XML
+    tinyxml2::XMLDocument map;
+    tinyxml2::XMLError eResult = map.LoadFile("../res/practice_level.xml");
+    if (eResult != tinyxml2::XML_SUCCESS) {
+        std::cout << "error loading file" << std::endl;
```

Comment:
I would put a return statement here so that way the function exits when it fails

Reply:
Done